templates/base.html

```diff
@@ -61,7 +61,12 @@
                             <span class="icon-bar"></span>
                             <span class="icon-bar"></span>
                         </a>
-                        <a class="brand" href="{{ SITEURL }}/"><span class=site-name>{{ SITENAME }}</span></a>
+                        <a class="brand" href="{{ SITEURL }}/">
+                            {% if DISPLAY_BRAND_LOGO %}
+                                <img src="{{ DISPLAY_LOGO_ON_TITLE }}" alt="brand-logo">
```

Comment:
If Using DISPLAY_BRAND_LOGO the IMG SRC should be the same variable, if not, it sets an undefined variable